python/src/dataset.rs

```diff
@@ -1820,6 +1822,19 @@ pub fn get_write_params(options: &Bound<'_, PyDict>) -> PyResult<Option<WritePar
             });
         }
 
+        if let Some(writer_options) =
+            get_dict_opt::<HashMap<String, String>>(options, "file_writer_options")?
+        {
+            let mut file_writer_options = FileWriterOptions::default();
+            if let Some(max_page_bytes) = writer_options.get("max_page_bytes") {
+                file_writer_options.max_page_bytes = max_page_bytes.parse::<u64>().ok()
+            }
+            if let Some(data_cache_bytes) = writer_options.get("data_cache_bytes") {
+                file_writer_options.data_cache_bytes = data_cache_bytes.parse::<u64>().ok()
```

Comment:
Does `ok()` here mean we will silently ignore parse errors?  It might be better to raise an error to the user.

Reply:
OK, I add a warn here